### PR TITLE
[TG-196] [feat] Confirmation dialogues on deleting items

### PR DESCRIPTION
**Pull request by valakar**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/127" target="_blank">ReviewNinja

**Comment by stremann** [comment]
@valakar please follow the git naming convention provided on [wiki](https://github.com/garage-it/SmartHouse-frontend/wiki/Working-with-git#naming-convention) (e.g. [examples](https://github.com/garage-it/SmartHouse-frontend/commits/master))

File: src/app/shared/dialog/dialog-confirm.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<p>{{ title }}</p>
+<p>{{ message }}</p>
+<button type="button" md-raised-button
+        (click)="confirm()">{{ yes }}</button>
+<button type="button" md-button
+        *ngIf="no"
+        (click)="close()">{{ no }}</button>
```
Comment (piton13) [inline src/app/shared/dialog/dialog-confirm.html]: !fix don't obvious names of confirmation buttons

Comment (valakar) [inline src/app/shared/dialog/dialog-confirm.html]: !fixed renamed

**Pull request merged**